linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_ocredit/rule.yml

```
@@ -56,15 +56,21 @@ references:
     stigid@ubuntu2004: UBTU-20-010055
     vmmsrg: SRG-OS-000266-VMM-000940
 
-ocil_clause: 'ocredit is not found or not equal to or less than the required value'
+ocil_clause: 'ocredit is not found or is not equal to or less than the required value'
```

The latest RHEL8 STIG has `If the value of "ocredit" is a positive number or is commented out, this is a finding.`

I believe we can change to something similar, like:

```suggestion
ocil_clause: 'value of "ocredit" is a positive number or is commented out'
```

---

The text was reflecting the OVAL instructions. However, I am fine with this suggestion since it is more clear.